tools/cassandra/cqlclient.go

```diff
@@ -105,7 +105,7 @@ func NewCQLClient(cfg *CQLClientConfig) (*CqlClient, error) {
 		Keyspace:     cfg.Keyspace,
 		TLS:          cfg.TLS,
 		Timeout:      time.Duration(cfg.Timeout) * time.Second,
-		ProtoVersion: cqlProtoVersion,
+		ProtoVersion: cfg.ProtoVersion,
```

Comment:
- Is this field initialized in `newCQLClientConfig`?
- Shall we provide a flag for specifying the value in the cassandra tool CLI, which is different from the cadence CLI?

Reply:
1) It is now :)
2) Is that in `tools/common/schema/types.go` ? If that's the case, that's done